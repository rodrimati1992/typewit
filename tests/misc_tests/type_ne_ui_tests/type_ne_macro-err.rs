const _: () = {
    let _ = typewit::type_ne!{u8, u8};
    let _ = typewit::type_ne!{<T> T, T};
};

fn main() {}