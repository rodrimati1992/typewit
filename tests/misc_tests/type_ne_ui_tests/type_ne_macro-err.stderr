error[E0119]: conflicting implementations of trait `RevTypeFn<u8>` for type `_::__TypeNeParameterizer`
 --> tests/misc_tests/type_ne_ui_tests/type_ne_macro-err.rs:2:13
  |
2 |     let _ = typewit::type_ne!{u8, u8};
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^
  |             |
  |             first implementation here
  |             conflicting implementation for `_::__TypeNeParameterizer`
  |
  = note: this error originates in the macro `$crate::__impl_with_span` which comes from the expansion of the macro `typewit::type_ne` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0119]: conflicting implementations of trait `RevTypeFn<_>` for type `_::__TypeNeParameterizer<_>`
 --> tests/misc_tests/type_ne_ui_tests/type_ne_macro-err.rs:3:13
  |
3 |     let _ = typewit::type_ne!{<T> T, T};
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |             |
  |             first implementation here
  |             conflicting implementation for `_::__TypeNeParameterizer<_>`
  |
  = note: this error originates in the macro `$crate::__impl_with_span` which comes from the expansion of the macro `typewit::type_ne` (in Nightly builds, run with -Z macro-backtrace for more info)
